### PR TITLE
Added a button that curls on hover

### PR DESCRIPTION
**Pull request by wisetok12680**
Added a button that curls on hover

Issue:
No Issues

### CODE REVIEW

File: src/components/animated/_papercurl.less [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    position: absolute;
+    right: 0;
+    transition-duration: 0.3s;
+    transition-property: width, height;
+    width: 0;
+    border-color: white;
+  }
+  &:hover::before,
+  &:focus::before,
+  &:active::before {
```
Comment (shahednasser) [inline src/components/animated/_papercurl.less:27]: You can nest before inside as well. The code should be something like this:

```
&:hover,
  &:focus,
  &:active {
   &:before {
      //code here
    }
}
```

**Review by shahednasser — CHANGES_REQUESTED** [review]
You've done the nesting well! I just need you to make this last change

File: src/components/animated/_papercurl.less (hunk 1 above)
Comment (wisetok12680) [reply src/components/animated/_papercurl.less:27]: Ok

Comment (wisetok12680) [reply src/components/animated/_papercurl.less]: I just made the requested change.
Hope everything is alright now.

**Comment by shahednasser** [comment]
Great work thank you!

@all-contributors please add @wisetok12680 for code and idea

**Pull request merged**